.github/workflows/ci.yml

```diff
@@ -18,8 +18,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest]
-        python-version: ['3.8', '3.11', '3.12']
+        os: [ubuntu-20.04]
```

Comment:
Why are we reverting the os version?
https://github.com/openedx/public-engineering/issues/278

Reply:
Agree with the Ubuntu comment, but isn't the line below what you were requesting vis-a-via python?

`python-version: ['3.11', '3.12']`

Reply:
@e0d, I'm afraid I don't understand this message. Since we are dropping Python 3.8 support here, we should update the version of this XBlock (in `image_explorer/__init__.py`) and [mark it as a breaking change](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html) because this XBlock will no longer work with pre-Redwood releases.

Reply:
it might've happen mistakenly while rebasing. i'll revert it back